faust.config.js

```diff
@@ -2,11 +2,28 @@ import { setConfig } from '@faustwp/core';
 import templates from './wp-templates';
 import possibleTypes from './possibleTypes.json';
 
+class ProjectTemplatePlugin {
```

Comment:
Really a workaround as ACM does not set the [has_archive](https://developer.wordpress.org/reference/functions/register_post_type/#has_archive)  yet...